build_tools/python/e2e_test_framework/definitions/common_definitions.py

```diff
@@ -56,15 +56,16 @@ class DeviceArchitecture(Enum):
 
 @dataclass(frozen=True)
 class PlatformInfo(object):
-  """Platform information of a device."""
+  """Platform information of a host."""
   os: str
+  architecture: str
 
 
-class DevicePlatform(Enum):
-  """Predefined device platform information."""
+class HostPlatform(Enum):
+  """Predefined host platform information."""
 
-  GENERIC_LINUX = PlatformInfo("linux")
-  GENERIC_ANDROID = PlatformInfo("android")
+  GENERIC_LINUX = PlatformInfo("linux", "x86_64")
+  GENERIC_ANDROID = PlatformInfo("android", "armv8.2-a")
```

Comment:
These aren't generic though!

```suggestion
  LINUX_X86_64 = PlatformInfo("linux", "x86_64")
  ANDROID_ARM_64 = PlatformInfo("android", "armv8.2-a")
```

Reply:
SG. Done.